packages/bundler-plugin-core/src/sentry/releasePipeline.ts

```diff
@@ -36,10 +39,13 @@ export async function cleanArtifacts(
 
   const span = addSpanToTransaction(ctx, "function.plugin.clean_artifacts");
 
-  await ctx.cli.releases.execute(["releases", "files", releaseName, "delete", "--all"], true);
+  try {
+    await ctx.cli.releases.execute(["releases", "files", releaseName, "delete", "--all"], true);
+  } finally {
+    span?.finish();
+  }
```

Comment:
Don't we put `span` onto the scope in `addSpanToTransaction`? In case we do, can we finish the span in the try/catch/finally that wraps the entire release creation pipeline?

Reply:
I had that before but it felt very blanket-approachy. I like this more because it is more explicit (though admittedly repetitive).

Reply:
Fair enough, was just a thought 😅